cl_dll/hud_oldscoreboard.cpp

```diff
@@ -0,0 +1,300 @@
+#include "hud.h"
+#include "cl_util.h"
+#include <string.h>
+#include <stdio.h>
+#include <demo_api.h>
+#include "vgui_TeamFortressViewport.h"
+#include "hud_oldscoreboard.h"
+#include "vgui_ScorePanel.h"
+
+// Y positions
+// Those who play on killed-off 32bit Apple don't deserve old_scoreboard looking good
+// as I am too lazy to build & test in a MacOS, sorry.
+#ifdef LINUX
+	#define ROW_RANGE_MIN (gHUD.m_scrinfo.iCharHeight + 3)
+
+	/* The scoreboard's default width is set to 380 on Linux
+	 * since it doesn't look good (Ping/loss doesn't fit and overlaps "Deaths" etc.)
+	 * on the (original) 320 default, like it does on Windows,
+	 * since the engine's Trebuchet MS is bigger than on Windows
+	 * WINDOWS = Trebuchet MS = 9x8
+	 * LINUX = Trebuchet MS = 11x10
+	 */
+	#define DEFAULT_WIDTH "380"
+	#define DEFAULT_WIDTH_NUM 380
+#else
+	#define ROW_RANGE_MIN (gHUD.m_scrinfo.iCharHeight + 2)
+	#define DEFAULT_WIDTH "320"
+	#define DEFAULT_WIDTH_NUM 320
```

Comment:
As far as I can see the _NUM is no longer used.

Reply:
Correct. Got rid of it.